diff --git a/lib/app_router.dart b/lib/app_router.dart
--- a/lib/app_router.dart
+++ b/lib/app_router.dart
@@ -560,7 +560,7 @@
-              'La page "${state.uri}" n\\'existe pas.',
+              'La page "${state.uri}" n'existe pas.',
@@ -572,7 +572,7 @@
-                'Retour à l\\'accueil'
+                'Retour à l'accueil'
@@ -188,9 +188,11 @@
-            gardenName: gardenName);
+            gardenName: gardenName,
+          );
@@ -440,10 +442,12 @@
-            plantId: plantId,
-            bedId: bedId);
+            plantId: plantId,
+            bedId: bedId,
+          );
@@ -451,7 +455,9 @@
-            gardenId: gardenId);
+            gardenId: gardenId,
+          );
